specs/core/0_beacon-chain.md

```diff
@@ -1699,7 +1701,9 @@ For each `attestation` in `block.body.attestations`:
 ```
 
 * [TO BE REMOVED IN PHASE 1] Verify that `attestation.data.crosslink_data_root == ZERO_HASH`.
-* Append `PendingAttestation(data=attestation.data, aggregation_bitfield=attestation.aggregation_bitfield, custody_bitfield=attestation.custody_bitfield, inclusion_slot=state.slot)` to `state.latest_attestations`.
+* Let `pending_attestation = PendingAttestation(data=attestation.data, aggregation_bitfield=attestation.aggregation_bitfield, custody_bitfield=attestation.custody_bitfield, inclusion_slot=state.slot)`.
+* Append `pending_attestation` to `state.previous_epoch_attestations` if `slot_to_epoch(attestation.data.slot) == get_previous_epoch(state)`.
+* Append `pending_attestation` to `state.current_epoch_attestations` if `slot_to_epoch(attestation.data.slot) == get_current_epoch(state)`.
```

Comment:
Possible corner case: the genesis epoch where at present the previous and current epochs are the same.

Honestly I think the cleanest ideal solution is to remove that exception and make previous always equal current-1...

Reply:
Agreed that current-1 always is ideal. Is that possible @djrtwo?

Reply:
I've gone back and forth on this a few times. I just did a read through and _I think_ it will just work. Because no validators exists in the first "previous epoch" then all of the epoch transition operations on previous epoch essentially become a null op. Depending on how you choose to process things, there is a small chance a client might divide by zero during the first epoch transition, but this would found very early on in the dev process.